### ISSUE TITLE
New categories

### ISSUE THREAD

**Issue by Antonio-52** [organicmaps/organicmaps#6312]
When I try to add a place on the map, many times there is no suitable category to apply.
For example, the following categories are missing and should be added to allow precise categorization:
- Mine (in Italian, Miniera)
- Cave (in Italian, Cava)
- Glacial erratic (in Italian, Masso erratico)

I work with Android app 2023.09.22-27-Google

Thank you!

**Comment by patepelo** [comment]
Thanks for giving feedback. 

It might be possible that Organic Maps will include these categories for adding in the future but it is not intended to become an editor app. For these I recommend using Osm Go!, Street Complete and Every Door, or even Vespuccii for more advanced editing if you are on android. Or just add these elements on OpenStreetMap.org with the we editor on desktop. You will find all those edits in the subsequent map update. 

Have you tried one of those before?

Would you still prefer to add them from OM?

**Comment by biodranik** [comment]
Isn't cave already present in the Editor?

**Comment by Antonio-52** [comment]
Sorry.
Cave there I'd in categories (Grotta, in Italian)

I means "Quarry", that translates "Cava" in Italian.

Please apologize me.

--
Antonio Angelo

Inviato da Proton Mail mobile

-------- Messaggio originale --------
Il 14 ott 2023, 20:09, Alexander Borsuk ha scritto:

**Comment by Antonio-52** [comment]
No, I do not know the applications you listed.

I think that new categories could be added in OM, but published after a check.

Thanks.

--
Antonio Angelo

Inviato da Proton Mail mobile

-------- Messaggio originale --------
Il 14 ott 2023, 20:04, Anton ha scritto:

**Comment by Antonio-52** [comment]
No, I work in Italian language but I cannot find nothing similar to Cave (cab in Italian language).

Another question:
In Italy all paths have a numeric code assigned by CAI (Club Alpino Italiano).
Is there a way to display (or add in some way) this numeric code to each path?

Thank you.

--
Antonio Angelo

Inviato da Proton Mail mobile

-------- Messaggio originale --------
Il 14 ott 2023, 20:09, Alexander Borsuk ha scritto:

**Comment by Antonio-52** [comment]
Hi!
My proposal does not refer to so called sac_scale, that is a classification of type/difficulty of the track, but the CAI numeric code reported on all maps.

Users could be enabled to label tracks they know with specific CAI numbers.

--
Antonio Angelo

Inviato da Proton Mail mobile

-------- Messaggio originale --------
Il 15 ott 2023, 05:33, Brad Baker ha scritto:

**Comment by biodranik** [comment]
Let's create a separate issue for the hike classification. Or did we already have one?

**Comment by Antonio-52** [comment]
I do not create any issue for the hike classification.
I think it is better to create a new one.

--
Antonio Angelo

Inviato da Proton Mail mobile

-------- Messaggio originale --------
Il 15 ott 2023, 21:52, Alexander Borsuk ha scritto:

**Comment by Antonio-52** [comment]
> Sorry about that! It looks like OSM uses the `ref` tag for CAI numbers: https://wiki.openstreetmap.org/wiki/IT:CAI (I don't speak Italian though!)

I looked at [https://www.openstreetmap.org/relation/7015292#map=15/45.6829/7.9572](https://www.openstreetmap.org/relation/7015292#map=15/45.6829/7.9572) and, if I understand, 'ref' tag is correctly set to E40 for one of hiking route of my interest.
Actually, does OrganicMaps show ref values for hiking route?
If no, is it possible to display this?
Thank you.

**Comment by pastk** [comment]
> Sorry. Cave there I'd in categories (Grotta, in Italian) I means "Quarry", that translates "Cava" in Italian. 

`landuse=quarry` is predominantly an areal feature (just 13% of all mapped quarries are nodes).

Mine.. `historic=mine`? Good to add along with other missing "historic" features.

"Glacial erratic (in Italian, Masso erratico)"
sorry I don't understand what is it.. what is an OSM tag? or could you give an example coordinates of such a feature on OSM?

**Comment by pastk** [comment]
OK I guess you meant [`geological=glacial_erratic`](https://wiki.openstreetmap.org/wiki/Tag:geological%3Dglacial_erratic). Its 250 uses only, I suggest to map as `natural=rock` ("Rock", "Roccia") instead.

And we should add `natural=stone` (55k uses) to OM as a synonym to `natural=rock`.

**Comment by Antonio-52** [comment]
Well, I suggested "glacial erratic" because they are quite specific: not simply "rocks".
The denomination (tag?) geological=glacial_erratic seems to me much more useful because has the attribute "geological" that enhance the "geological" interest of the occurrence.

So I do prefer geological=glacial_erratic instead of natural=stone because the former does not add any clarification to the occurrence.

**Comment by biodranik** [comment]
Most of our users never heard about OSM. And they may be scared by a long list of all possible types of objects. Does it make sense to focus only on the most basic ones, easy to understand by our grandparents, and leave any advanced cases for other editors (or allow leaving a note)?

**Comment by Antonio-52** [comment]
Doing so, OSM will never grow.
In Europe, in all Alps area we have a lot of occurrences of erratic rocks.
If someone doesn't know what erratic mean, simply will not use that tag.

--
Antonio Angelo

Inviato con l'email sicura [Proton Mail](https://proton.me/).

lunedì 1 aprile 2024 01:43, Alexander Borsuk ***@***.***> ha scritto:
